client/src/Pages/SignUpPage.js

```diff
@@ -0,0 +1,11 @@
+import CreateAccount from '../Components/CreateAccount/CreateAccount'
+
+const SignUpPage = () => {
+  return (
+    <div className='page-container'>
+      <CreateAccount />
```

Comment:
Will SignUpPage have more components than just this later on? Myabe slightly pointless to have it organized this way.

Reply:
yes.

Reply:
A longer answer is that CreateAccount is a single component that handles its internal styling and positioning, but nothing else outside of it. It has no bearing on the positioning within the larger page. The point of the page components (not just SignUpPage but every other page as well) is to handle the layout and positioning of things that are not contained within the component.

Keeping the CSS decoupled like this makes it much easier to style components without worrying about the contents of the rest of the page being pushed around or messed with.

This seems like a simpler, cleaner, and more maintainable solution than, say, adding a margin to the CreateAccount component to position it properly. This way, when we want to change the layout of the SignUpPage, we have a single consolidated CSS file to do so without having to delicately jump around all the CSS files that correspond to components that are only imported into the SignUpPage.